recipe/meta.yaml

```diff
@@ -1,15 +1,17 @@
-{% set version = "1.5.4" %}
+{% set version = "1.5.5" %}
 
 package:
   name: pkgconfig
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pkgconfig/pkgconfig-{{ version }}.tar.gz
-  sha256: c34503829fd226822fd93c902b1cf275516908a023a24be0a02ba687f3a00399
+  sha256: deb4163ef11f75b520d822d9505c1f462761b4309b1bb713d08689759ea8b899
 
 build:
   number: 0
+  # skip s390x due to missing the package poetry for the s390x architecture
+  skip: True  # [py<36 or s390x]
```

Comment:
Does poetry not build for s390x? We should not skip a platform just because a dependency is not available; we should just build that dependency instead.

Also, I don't see any evidence that this package is not supported for Python <3.6.

Reply:
the package is now building for `s390x`